en/docs/quick-starts/android.md

```diff
@@ -83,76 +133,70 @@ Create a `oidc_config.json` file inside the `res/raw` folder. Add the following
 Example:
 
 ```json
+{
 "client_id": "rs5ww91iychg9JN0DJGLMaxG2gha",
  "redirect_uri": "wso2sample://oauth2",
  "authorization_scope": "openid",
  "discovery_uri": "https://stgcloud.kubesb.com/t/example/oauth2/oidcdiscovery/.well-known/openid-configuration"
 }
 ```
 
-
 ### Login
 
-As the first step, you need to initialize SDK in the Activity#onCreate method of the Activity that you are using to
- log users into your app. 
-In this example, we will call it LoginActivity:
+- First, you need to initialize SDK object in your Activity that you are using to log users into your app. 
+
+- In this example, we will call it LoginActivity. After successful authentication user will be redirected to another
+ activity. Lets name it as UserInfoActivity.
 
 ```java
-    mLoginService = new DefaultLoginService(this);
+    LoginService mLoginService = new DefaultLoginService(this);
 ```
 
 
-#### Authorization.
+- Have a login button inside LoginActivity. Here the button id is referred as `login`.
 
-Have a login button inside LoginActivity. Call the `doAuthorization()` method 
- when the login button is clicked to call authorization flow.
+- Call the`doLogin()` method  when the login button is clicked to initiate authentication with
+ Identity Server.
  
-Have a login button inside LoginActivity. Call the `doAuthorization()` method 
-  when the login button is clicked to call authorization flow.
-  
-  `authorize(PendingIntent successIntent, PendingIntent failureIntent,
-              Boolean callUserInfo)`
+- When calling authorize method of LoginService, you have to create completionIntent, and
```

Comment:
You need to create `completionIntent`, and `cancelIntent` when calling the authorize method of `LoginService`.